.github/actions/end-to-end-tests/action.yaml

```diff
@@ -10,6 +10,10 @@ runs:
         shell: bash
         run: pnpm --filter e2e-${{ inputs.app }} run install:chromium
 
+      - name: Build dependencies for e2e tests
+        shell: bash
+        run: pnpm turbo build --filter e2e-base^...
+
       - name: Run end-to-end tests
```

Comment:
Need to build dependencies of e2e-base package so we can actually use things like @kadena/client. I honestly have no idea how this could've ever worked before....